package.json

```diff
@@ -32,6 +32,8 @@
     "plan:prod": "terraform -chdir=infra workspace select prod && terraform -chdir=infra plan",
     "plan:staging": "terraform -chdir=infra workspace select staging && terraform -chdir=infra plan",
     "prod": "terraform -chdir=infra workspace select prod && npm run cache:clear",
+    "refill:alfajores": "NODE_ENV=development npx ts-node ./src/refill-relayers.ts alfajores",
```

Comment:
why do we need NODE_ENV here?

Reply:
it's a required var in the .[env config](https://github.com/mento-protocol/oracle-relayer/blob/main/src/config.ts#L6) so without it the script complains that the var is missing